backend/base/urls.py

```diff
@@ -3,18 +3,32 @@
 from django.conf.urls import url, include
 from . import views
 from django.contrib.auth import views as auth_views
+from django.contrib import admin
 
 from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+
+
+admin.autodiscover()
```

Comment:
Was this moved here from somewhere else? I truthfully don't entirely know what this does.

Reply:
vestigial from debug early in the process. removing